src/DynamoPackages/PackageLoader.cs

```diff
@@ -207,13 +207,26 @@ internal void TryLoadPackageIntoLibrary(Package package)
                 package.Loaded = true;
                 this.PackgeLoaded?.Invoke(package);
             }
+            catch (CustomNodePackageLoadException e)
+            {
+                Package originalPackage =
```

Comment:
@QilongTang what do you think - should we log this to analytics?

Reply:
Worth! But are you thinking about public APIs for extention author to use or internal call only managed by our team?

Reply:
am thinking about the regular log Analytics calls for events - I'm not sure if this would already be logged a custom exception type or we only log uncaught exceptions?